addons/mail/static/tests/tours/discuss_public_tour.js

```diff
@@ -9,12 +9,12 @@ tour.register(
     },
     [
         {
-            trigger: ".o_DiscussPublicView",
-            extraTrigger: ".o_ThreadView",
+            trigger: ".o-mail-discuss-public",
+            extraTrigger: ".o-mail-discuss",
```

Comment:
I would expect `o_ThreadView` to become `o-mail-thread` instead.

Reply:
will do this in one of the following pr needed for rebasing